docs/tutorials/Linux Ubuntu 16.04 Tutorial 0 - Hello, world!

```diff
@@ -0,0 +1,84 @@
+Linux Ubuntu 16.04 Tutorial 0 - Hello, world!
+==========================
+
+In this tutorial, you'll take a really simple "Hello, world!" program written in
+Python, convert it into a working Ubuntu Linux project.
```

Comment:
Is there anything completely Ubuntu specific here? AFAIK, this set of instructions should work for *any* Linux; so I'd rather label this as a "Linux" tutorial, with a note early on that says "this has been tested on Ubuntu 16.04, but should work on other Unixes"